libraries/AP_Baro/AP_Baro.h

```diff
@@ -240,6 +244,19 @@ class AP_Baro
         PROBE_MSP   =(1<<12),
     };
     
+#if HAL_BARO_WIND_COMP_ENABLED
+    class WindCoeff {
+    public:
+        static const struct AP_Param::GroupInfo var_info[];
+
+        AP_Int8  enable; // enable compensation for this barometer
+        AP_Float xp;     // ratio of static pressure rise to dynamic pressure when flying forwards
```

Comment:
At some point it would be nice to rename these variables to match the parameter names (which are easier to understand I think)

Reply:
Noted.